contracts/wrapped_zil.scilla

```diff
@@ -0,0 +1,266 @@
+scilla_version 0
```

Comment:
@sam-zilliqa I think this was meant for another pull request, right? This is not needed for brand collection/commission contract?